### PR TITLE
Add initial version of geolocator

### PR DESCRIPTION
**Pull request by divy4**

### CODE REVIEW

File: plane_spotter/geolocator.py [hunk 1]
```diff
@@ -1,15 +1,87 @@
...
-    pass
+    self.__airports = []
+    # Add airport dict for each row in airport_code_file
+    with open(airport_code_file, 'r') as file:
+      reader = csv.reader(file)
+      headers = next(reader)
+      for line in reader:
+        airport = {}
+        # Convert row to dict
+        for key, value in zip(headers, line):
```
Comment (LandonTClipp) [inline plane_spotter/geolocator.py]: You can use a `DictReader` to get the key-val pair for each row with headers already in it.

https://docs.python.org/3/library/csv.html#csv.DictReader

File: plane_spotter/geolocator.py [hunk 2]
```diff
@@ -1,15 +1,87 @@
...
+          airport[key] = value
+        # Add it if it's not a blacklisted type
+        if airport['type'] not in AIRPORT_TYPE_BLACKLIST:
+          coordinates = airport['coordinates'].split(', ')
+          airport['coordinates'] = (float(coordinates[0]), float(coordinates[1]))
+          self.__airports.append(airport)
+    if len(self.__airports) == 0:
+      raise Exception('No airports loaded.')
```
Comment (LandonTClipp) [inline plane_spotter/geolocator.py]: Maybe `RuntimeError` is more appropriate

File: plane_spotter/geolocator.py [hunk 3]
```diff
@@ -1,15 +1,87 @@
...
 
+import csv
+import haversine
+
+AIRPORT_TYPE_BLACKLIST = set(('balloonport', 'closed', 'heliport', 'seaplane_base'))
+
 class Geolocator:
   ''' Translates coordinates to the closest airport. '''
 
   def __init__(self, airport_code_file: str='plane_spotter/data/airport-codes.csv') -> 'Geolocator':
```
Comment (LandonTClipp) [inline plane_spotter/geolocator.py]: I would actually make `airport_code_file` a `pathlib.Path` type now that I think of it.

File: plane_spotter/geolocator.py (hunk 1 above)
Comment (divy4) [reply plane_spotter/geolocator.py]: Love it.

File: plane_spotter/geolocator.py (hunk 2 above)
Comment (divy4) [reply plane_spotter/geolocator.py]: Good point!

File: plane_spotter/geolocator.py (hunk 3 above)
Comment (divy4) [reply plane_spotter/geolocator.py]: So much to relearn in Python...

File: plane_spotter/geolocator.py [hunk 4]
```diff
@@ -1,15 +1,90 @@
...
+            "expected": "Ronald Reagan Washington National Airport",
+        },
+        {
+            "coordinates": (38.89804315682689, -77.0040050557741),
+            "dist": 1,
+            "expected": None,
+        },
+    ]
+
+    # Yes, I know there's testing frameworks. I just wanted something simple.
```
Comment (LandonTClipp) [inline plane_spotter/geolocator.py:78]: 👍🏻

**Review by LandonTClipp — APPROVED** [review]
Looks good!

**Pull request merged**